pkg/option/config.go

```diff
@@ -2136,6 +2136,13 @@ func (c *DaemonConfig) SetIPv4NativeRoutingCIDR(cidr *cidr.CIDR) {
 	c.ConfigPatchMutex.Unlock()
 }
 
+// IPv4NativeRoutingCIDRRLocked returns the native routing CIDR if configured,
+// c.ConfigPatchMutex must be held when calling this function.
+func (c *DaemonConfig) IPv4NativeRoutingCIDRRLocked() (cidr *cidr.CIDR) {
+	cidr = c.ipv4NativeRoutingCIDR
+	return
+}
```

Comment:
We can simply return `c.ipv4NativeRoutingCIDR` without needing to make a copy of the pointer. This gets rid of the named return value.

If the intent was to protect `c.ipv4NativeRoutingCIDR`, then a deep copy is required.